docs/user/dql/functions.rst

```diff
@@ -3839,6 +3839,48 @@ Another example to show how to set custom values for the optional parameters::
     | 1    | The House at Pooh Corner | Alan Alexander Milne |
     +------+--------------------------+----------------------+
 
+SCORE
+------------
+
+Description
+>>>>>>>>>>>
+
+``score(relevance_expression[, boost])``
+``score_query(relevance_expression[, boost])``
+``scorequery(relevance_expression[, boost])``
```

Comment:
Looking into it, I do not think boost is multiplier here.

Consider [Maker.java](https://github.com/opensearch-project/sql/blob/bc3934611b5a60ebc981c18b180b58883bf6be55/legacy/src/main/java/org/opensearch/sql/legacy/query/maker/Maker.java#L143) -- it creates a [ConstantScoreQueryBuilder](https://github.com/opensearch-project/OpenSearch/blob/5f8193021215cd6979fde66474ec8d74d32ac91a/server/src/main/java/org/opensearch/index/query/ConstantScoreQueryBuilder.java#L54).

Reply:
For posterity: during development it was agreed that using `x` in `score(..., x)` as boost multiplier was more useful than current implementation.